custom_connectors/basic_auth/stripe_connector.rb

```diff
@@ -0,0 +1,920 @@
+{
+  title: "Stripe SDK",
+
+  connection: {
+    fields: [
+      {
+        name: "api_key",
+        control_type: "password",
+        optional: false,
+        label: "API Key",
+        hint: "Get your API key <a href='https://dashboard.stripe.com/" \
+          "account/apikeys' target='_blank'>here</a>"
+      },
+      {
+        name: "api_version",
+        optional: true
```

Comment:
what are the available versions? How is the user suppose to know? How about a default version?